src/frequenz/client/dispatch/_client.py

```diff
@@ -37,23 +40,43 @@
 )
 
 # pylint: enable=no-name-in-module
+DEFAULT_DISPATCH_PORT = 50051
 
 
-class Client:
+class Client(BaseApiClient[dispatch_pb2_grpc.MicrogridDispatchServiceStub]):
     """Dispatch API client."""
 
     def __init__(
-        self, *, grpc_channel: grpc.aio.Channel, svc_addr: str, key: str
+        self,
+        *,
+        svc_addr: str,
```

Comment:
Sorry I missed this before, but could you use `service_url` instead, like the base client, so we have a consistent interface across all the different API clients?
```suggestion
        server_url: str,
```